docs/machine-learning/resources/azure-training-concepts-model-builder.md

```diff
@@ -59,7 +59,23 @@ Once training is complete, two projects are added to your solution with the foll
   - bestModel.onnx: A serialized version of the model in Open Neural Network Exchange (ONNX) format. ONNX is an open source format for AI models that supports interoperability between frameworks like ML.NET, PyTorch and TensorFlow.
   - bestModelMap.json: A list of categories used when making predictions to map the model output to a text category.
   - MLModel.zip: A serialized version of the ML.NET prediction pipeline that uses the serialized version of the model *bestModel.onnx* to make predictions and maps outputs using the `bestModelMap.json` file.
-  
+
+## Consume the model
+
+The `ModelInput` and `ModelOutput` classes in the *Model* project define the schema of the model's expected input and output respectively.
+
+In an image classification scenario, the `ModelInput` contains two columns:
+
+- `ImageSource`: The string path of the image location.
+- `Label`: The actual category the image belongs to.
```

Comment:
We could say something like: "Label is only used as an input when training", to further clarify your statement below that Label is not needed for making predictions

Reply:
resolved